server/routes/api.js

```diff
@@ -0,0 +1,135 @@
+const express = require('express')
+const WeatherApi = require('./services')
+const weatherApi = new WeatherApi
+const router = express.Router()
+const moment = require('moment')
+const City = require('../models/city')
+
+const updateDbHourly = async function () {
+    const result = await City.find({}, { _id: 0, openWeatherId: 1 })
+    const citiesIds = []
+    result.forEach(id => citiesIds.push(id.openWeatherId))
```

Comment:
instead of `.forEach()` you can use `.map()` this will create the array for you and you do not need to `.push()`. In general using `.map()` is better.